apps/docs/src/.vitepress/config.ts

```diff
@@ -24,40 +24,68 @@ export default defineConfig({
     },
     lastUpdated: {}, // needed to show the last updated text with default settings
     nav: [
-      { text: "Getting Started", link: "/getting-started" },
+      { text: "Brand", link: "/brand/team" },
+      { text: "Basics", link: "/basics/" },
+      { text: "Tokens", link: "/tokens/" },
+      { text: "Development", link: "/development/getting-started" },
       { text: "Report a bug", link: packageJson.bugs.url },
       { text: "Q&A", link: "https://github.com/schwarzit/onyx/discussions/categories/q-a" },
     ],
     socialLinks: [{ icon: "github", link: packageJson.repository.url }],
-    sidebar: [
-      {
-        text: "Introduction",
-        collapsed: false,
-        items: [
-          { text: "Getting Started", link: "/getting-started" },
-          { text: "i18n", link: "/i18n/" },
-          { text: "Typography", link: "/typography" },
-          { text: "Colors", link: "/colors" },
-          { text: "The Team", link: "/team" },
-        ],
-      },
-      {
-        text: "Components",
-        base: "/components",
-        collapsed: false,
-        items: getComponents().map((name) => ({ text: name, link: `/${name}` })),
-      },
-      {
-        text: "Other onyx npm packages",
-        base: "/packages",
-        collapsed: false,
-        items: [
-          { text: "Figma utilities", link: "/figma-utils" },
-          { text: "Headless composables", link: "/headless" },
-          { text: "Storybook utilities", link: "/storybook-utils" },
-          { text: "VitePress theme", link: "/vitepress-theme" },
-        ],
-      },
-    ],
+    sidebar: {
+      "/brand": [
+        {
+          items: [{ text: "The Team", link: "/brand/team" }],
+        },
+      ],
+      "/basics": [
+        {
+          text: "Foundations",
+          collapsed: false,
+          base: "/basics",
+          items: [{ text: "Introduction", link: "/" }].sort((a, b) => a.text.localeCompare(b.text)),
```

Comment:
```suggestion
          items: [{ text: "Introduction", link: "/" }],
```

is this a leftover or is that a preparation for Jannicks PR?

Reply:
Preparation but I removed it so it will be added in this PR